lib/influxer/config.rb

```diff
@@ -28,15 +28,28 @@ class Config < Anyway::Config
                 :denormalize,
                 :udp,
                 :async,
+                :cache_enabled,
+                :cache,
                 database: "db",
                 time_precision: "ns",
-                cache: false,
                 time_duration_suffix_enabled: false
 
     def load(*)
       super
-      # we want pass @cache value as options to cache store, so we want it to be a Hash
-      @cache = {}.with_indifferent_access if @cache == true
+      if cache_enabled.nil?
+        self.cache_enabled = cache_enabled_value
+      end
+    end
+
+    def cache=(value)
+      super
+      self.cache_enabled = cache_enabled_value
+    end
+
+    private
+
+    def cache_enabled_value
+      !(cache == false || cache.nil?)
```

Comment:
Isn't it equal to `!!cache`?

Reply:
yes, usually rubocop marks it as wrong 😄